molecule/delegated/tests/trivy/redhat.py

```diff
@@ -0,0 +1,72 @@
+import pytest
+
+from ..util.util import (
+    get_ansible,
+    get_variable,
+    get_from_url,
+    extract_url_from_variable,
+    get_centos_repo_key,
+)
+
+testinfra_runner, testinfra_hosts = get_ansible()
+
+
+def check_ansible_os_family(host):
+    if get_variable(host, "ansible_os_family", True) != "RedHat":
+        pytest.skip("ansible_os_family mismatch")
+
+
+def test_package(host):
+    """Check if the packages are installed."""
+    check_ansible_os_family(host)
+
+    if get_variable(host, "trivy_configure_repository"):
+        pkg_name = "yum-utils"
+        pkg = host.package(pkg_name)
+        assert pkg.is_installed
```

Comment:
We dont need to test for the packages that were needed to install the actual packages

Reply:
ok